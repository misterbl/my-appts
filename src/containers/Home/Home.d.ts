// import { RouteComponentProps, StaticContext } from 'react-router';

// export interface IHome extends RouteComponentProps<unknown, StaticContext>
