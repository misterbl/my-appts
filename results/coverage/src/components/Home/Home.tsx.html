<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/Home/Home.tsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/components/Home</a> Home.tsx
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/17</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/14</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40</td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js"><span class="cstat-no" title="statement not covered" >import * as React from 'react';</span>
<span class="cstat-no" title="statement not covered" >import { FormattedMessage } from 'react-intl';</span>
<span class="cstat-no" title="statement not covered" >import { withRouter } from 'react-router-dom';</span>
import { RouteComponentProps, StaticContext } from 'react-router';
<span class="cstat-no" title="statement not covered" >import ROUTES from '../../consts/routes';</span>
&nbsp;
<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >export class <span class="cstat-no" title="statement not covered" >Home </span>extends <span class="cstat-no" title="statement not covered" >React.Component&lt;</span></span></span></span>
  RouteComponentProps&lt;any, StaticContext&gt;
&gt; {
<span class="cstat-no" title="statement not covered" >  pushToSignIn = <span class="fstat-no" title="function not covered" >() =&gt; {</span></span>
<span class="cstat-no" title="statement not covered" >    this.props.history.push('/signin');</span>
  };
&nbsp;
<span class="cstat-no" title="statement not covered" >  pushToRegister = <span class="fstat-no" title="function not covered" >() =&gt; {</span></span>
<span class="cstat-no" title="statement not covered" >    this.props.history.push(ROUTES.SIGN_IN);</span>
  };
<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >  render() {</span></span>
<span class="cstat-no" title="statement not covered" >    console.log(this.props.history.push);</span>
&nbsp;
<span class="cstat-no" title="statement not covered" >    return (</span>
      &lt;div className="text-center"&gt;
        &lt;img
          className="home-logo"
          src={require('../../assets/kidappi-logo.png')}
        /&gt;
        &lt;div className="space-around register-signin"&gt;
          &lt;button onClick={this.pushToSignIn} className="cta-button"&gt;
            &lt;FormattedMessage id="general|button|signin" /&gt;
          &lt;/button&gt;
          &lt;button onClick={this.pushToRegister} className="cta-button"&gt;
            &lt;FormattedMessage id="general|button|register" /&gt;
          &lt;/button&gt;
        &lt;/div&gt;
      &lt;/div&gt;
    );
  }
<span class="cstat-no" title="statement not covered" >}</span>
&nbsp;
<span class="cstat-no" title="statement not covered" >export default withRouter(Home);</span>
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Oct 23 2018 10:44:38 GMT+0100 (BST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
