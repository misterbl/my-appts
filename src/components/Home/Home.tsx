import * as React from 'react';
import { FormattedMessage } from 'react-intl';
import { withRouter } from 'react-router-dom';
import { RouteComponentProps, StaticContext } from 'react-router';
import ROUTES from '../../consts/routes';

export class Home extends React.Component<
  RouteComponentProps<any, StaticContext>
> {
  pushToSignIn = () => {
    this.props.history.push('/signin');
  };

  pushToRegister = () => {
    this.props.history.push(ROUTES.SIGN_IN);
  };
  render() {
    console.log(this.props.history.push);

    return (
      <div className="tc">
        <img
          className="home-logo"
          src={require('../../assets/kidappi-logo.png')}
        />
        <div className="space-around register-signin">
          <button onClick={this.pushToSignIn} className="cta-button">
            <FormattedMessage id="general|button|signin" />
          </button>
          <button onClick={this.pushToRegister} className="cta-button">
            <FormattedMessage id="general|button|register" />
          </button>
        </div>
      </div>
    );
  }
}

export default withRouter(Home);
