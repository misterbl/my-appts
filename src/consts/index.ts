import ROUTES from './routes';
import QUERIES from './queries';

export { ROUTES, QUERIES }