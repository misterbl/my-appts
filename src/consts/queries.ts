const queries = (args: any) => ({
  ADD_USER: `mutation {
    addUser (email: "${args.email}", avatar:"${args.avatar}" ) {
      _id
      firstName
      lastName
      avatar
      email
      address
      profileTitle
      profileDescription
     
    }
  }`,
  GET_USER: `mutation {
    getUser (email: "${args.email}") {
      _id
      firstName
      lastName
      avatar
      email
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability
    }
  }`,
  GET_USER_BY_ID: `mutation {
    getUserById (_id: "${args.id}") {
      _id
      firstName
      lastName
      avatar
      email
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
    }
  }`,
  GET_USER_INFO: `mutation {
    getUser (email: "${args.email}") {
      _id
      firstName
      lastName
      avatar
      email
      address
    }
  }`,
  GET_AD_INFO: `mutation {
    getUser (email: "${args.email}") {
      _id
      profileTitle
      profileDescription
    }
  }`,
  UPDATE_AD_INFO: `mutation {
    updateUser (_id:"${args._id}" profileTitle: "${
    args.profileTitle
  }", profileDescription: "${args.profileDescription}") {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{name, dob, information, school, gender} 
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
    }
  }`,
  UPDATE_PERSONAL_INFO: `mutation {
    updateUser (_id: "${args._id}", firstName: "${
    args.firstName
  }", lastName: "${args.lastName}", address: "${args.address}" ) {
      _id
      firstName
      lastName
      avatar
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}

    }
  }`,
  UPDATE_AVATAR: `mutation {
    updateUser (_id: "${args._id}", avatar:"${args.avatar}" ) {
      _id
      firstName
      lastName
      avatar
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}

    }
  }`,
  UPDATE_LOCATION: `mutation {
    updateUser (_id: "${args._id}", address: "${args.address}", lat: "${
    args.lat
  }", lng: "${args.lng}" ) {
      _id
      firstName
      lastName
      avatar
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}

    }
  }`,

  UPDATE_ABOUT_YOU: `mutation {
    updateUser (_id: "${args._id}", drivingLicense: ${
    args.drivingLicense
  }, nonSmoker: ${args.nonSmoker}, car:${args.car}) {
      _id
      firstName
      lastName
      avatar
      address
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability

    }
  }`,
  UPDATE_AVAILABILITY: `mutation {
    updateUser (_id: "${args._id}", availability: ${JSON.stringify(
    JSON.stringify(args.availability),
  )}) {
      _id
      firstName
      lastName
      avatar
      address
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
    }
  }`,
  UPDATE_UNAVAILABILITY: `mutation {
    updateUser (_id: "${args._id}", unavailability: ${JSON.stringify(
    args.unavailability,
  )}) {
      _id
      firstName
      lastName
      avatar
      address
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability
    }
  }`,
  ADD_CHILD: `mutation {
    updateUser (_id: "${args._id}", child: ${JSON.stringify(
    JSON.stringify(args.child),
  )}
) {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
    }
  }`,
  UPDATE_CHILD: `mutation {
    updateChild (_id: "${args._id}", child: ${JSON.stringify(
    JSON.stringify(args.child),
  )}
) {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability
    }
  }`,
  REMOVE_CHILD: `mutation {
    removeChild (_id: "${args._id}", child: ${JSON.stringify(
    JSON.stringify(args.child),
  )}
) {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
    }
  }`,
});

export default queries;
