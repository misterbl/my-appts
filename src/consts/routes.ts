const routes = {
  INDEX: '/',
  CARD: '/card',
  SIGN_IN: '/signin',
  REGISTER: '/register',
  DASHBOARD: './dashboard',
  INBOX: './inbox',
  FAVOURITE: './favourite',
  SEARCH: './search',
};

export default routes;
