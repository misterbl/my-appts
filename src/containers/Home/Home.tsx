import * as React from 'react';
import { FormattedMessage } from 'react-intl';
import { withRouter } from 'react-router-dom';
import { RouteComponentProps, StaticContext } from 'react-router';
import ROUTES from '../../consts/routes';

export class Home extends React.Component<
  RouteComponentProps<any, StaticContext>
> {
  pushToSignIn = () => {
    this.props.history.push(ROUTES.SIGN_IN);
  };

  pushToRegister = () => {
    this.props.history.push(ROUTES.REGISTER);
  };
  render() {
    return (
      <div className="flex flex-column vh-100 green-bg ph4 ph7-l ph6-m">
        {/* <img
          className="mt6 w-30"
          src={require('../../styles/assets/kidappi-logo.png')}
        /> */}
        <a className="kidappi white mt5 tc w-100">Kidappi</a>
        <button
          onClick={this.pushToSignIn}
          className="loginNext fw7 mt5 ttu di pv3 bn shadow-5 mh6-l mb4-l"
        >
          <FormattedMessage id="general|button|signin" />
        </button>
        <div className="text--line-through">
          <span className="spant3 ma0 tc white text--line-through--text">
            <FormattedMessage id="general|or" />
          </span>
        </div>
        <button
          onClick={this.pushToRegister}
          className="loginNext fw7 mt4 mt5-l ttu di pv3 bn shadow-5 mh6-l"
        >
          <FormattedMessage id="general|button|register" />
        </button>
      </div>
    );
  }
}

export default withRouter(Home);
