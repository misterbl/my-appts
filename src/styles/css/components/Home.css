.register-signin {
  position: absolute;
  width: 100%;
  bottom: 45px;
}

.home-logo {
  width: 130px;
  height: auto;
  margin-top: 100px;
}

/*# sourceMappingURL=Home.css.map */
