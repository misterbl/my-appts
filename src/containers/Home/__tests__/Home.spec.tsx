import * as React from 'react';
import { shallow } from 'enzyme';
import historyMock from '../../../testMocks/history.mock';
import { Home } from '../Home';
import 'regenerator-runtime/runtime';
import { configure } from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import { ROUTES } from '../../../consts';

configure({ adapter: new Adapter() });

describe('Home', () => {
  const props = {
    history: historyMock,
    location: historyMock.location,
    match: {
      params: {},
      isExact: true,
      path: '',
      url: '',
    },
    pushToSignIn: jest.fn(),
    pushToRegister: jest.fn(),
  };

  const wrapper = shallow(<Home {...props} />);
  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('should call pushToSignIn on button click', () => {
    wrapper
      .find('button')
      .at(0)
      .simulate('click');
    expect(props.history.push).toHaveBeenCalledWith(ROUTES.SIGN_IN);
  });
  it('should call pushToRegister on button click', () => {
    wrapper
      .find('button')
      .at(1)
      .simulate('click');
    expect(props.history.push).toHaveBeenCalledWith(ROUTES.REGISTER);
  });
});
