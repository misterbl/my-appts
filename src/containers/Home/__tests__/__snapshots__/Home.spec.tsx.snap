// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home
    history={
        Object {
            "action": "PUSH",
            "block": [Function],
            "createHref": [Function],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "length": 123,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "key": "",
              "pathname": "",
              "search": "",
              "state": "",
            },
            "push": [Function],
            "replace": [Function],
          }
    }
    location={
        Object {
            "hash": "",
            "key": "",
            "pathname": "",
            "search": "",
            "state": "",
          }
    }
    match={
        Object {
            "isExact": true,
            "params": Object {},
            "path": "",
            "url": "",
          }
    }
    pushToRegister={[Function]}
    pushToSignIn={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <a
          className="kidappi white mt5 tc w-100"
>
          Kidappi
</a>,
        <button
          className="loginNext fw7 mt5 ttu di pv3 bn shadow-5 mh6-l mb4-l"
          onClick={[Function]}
>
          <FormattedMessage
                    id="general|button|signin"
                    values={Object {}}
          />
</button>,
        <div
          className="text--line-through"
>
          <span
                    className="spant3 ma0 tc white text--line-through--text"
          >
                    <FormattedMessage
                              id="general|or"
                              values={Object {}}
                    />
          </span>
</div>,
        <button
          className="loginNext fw7 mt4 mt5-l ttu di pv3 bn shadow-5 mh6-l"
          onClick={[Function]}
>
          <FormattedMessage
                    id="general|button|register"
                    values={Object {}}
          />
</button>,
      ],
      "className": "flex flex-column vh-100 green-bg ph4 ph7-l ph6-m",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Kidappi",
          "className": "kidappi white mt5 tc w-100",
        },
        "ref": null,
        "rendered": "Kidappi",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <FormattedMessage
            id="general|button|signin"
            values={Object {}}
/>,
          "className": "loginNext fw7 mt5 ttu di pv3 bn shadow-5 mh6-l mb4-l",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "id": "general|button|signin",
            "values": Object {},
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "button",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <span
            className="spant3 ma0 tc white text--line-through--text"
>
            <FormattedMessage
                        id="general|or"
                        values={Object {}}
            />
</span>,
          "className": "text--line-through",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <FormattedMessage
              id="general|or"
              values={Object {}}
/>,
            "className": "spant3 ma0 tc white text--line-through--text",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "id": "general|or",
              "values": Object {},
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "span",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <FormattedMessage
            id="general|button|register"
            values={Object {}}
/>,
          "className": "loginNext fw7 mt4 mt5-l ttu di pv3 bn shadow-5 mh6-l",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "id": "general|button|register",
            "values": Object {},
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            className="kidappi white mt5 tc w-100"
>
            Kidappi
</a>,
          <button
            className="loginNext fw7 mt5 ttu di pv3 bn shadow-5 mh6-l mb4-l"
            onClick={[Function]}
>
            <FormattedMessage
                        id="general|button|signin"
                        values={Object {}}
            />
</button>,
          <div
            className="text--line-through"
>
            <span
                        className="spant3 ma0 tc white text--line-through--text"
            >
                        <FormattedMessage
                                    id="general|or"
                                    values={Object {}}
                        />
            </span>
</div>,
          <button
            className="loginNext fw7 mt4 mt5-l ttu di pv3 bn shadow-5 mh6-l"
            onClick={[Function]}
>
            <FormattedMessage
                        id="general|button|register"
                        values={Object {}}
            />
</button>,
        ],
        "className": "flex flex-column vh-100 green-bg ph4 ph7-l ph6-m",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Kidappi",
            "className": "kidappi white mt5 tc w-100",
          },
          "ref": null,
          "rendered": "Kidappi",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <FormattedMessage
              id="general|button|signin"
              values={Object {}}
/>,
            "className": "loginNext fw7 mt5 ttu di pv3 bn shadow-5 mh6-l mb4-l",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "id": "general|button|signin",
              "values": Object {},
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <span
              className="spant3 ma0 tc white text--line-through--text"
>
              <FormattedMessage
                            id="general|or"
                            values={Object {}}
              />
</span>,
            "className": "text--line-through",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <FormattedMessage
                id="general|or"
                values={Object {}}
/>,
              "className": "spant3 ma0 tc white text--line-through--text",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "id": "general|or",
                "values": Object {},
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "span",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <FormattedMessage
              id="general|button|register"
              values={Object {}}
/>,
            "className": "loginNext fw7 mt4 mt5-l ttu di pv3 bn shadow-5 mh6-l",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "id": "general|button|register",
              "values": Object {},
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
